docs/topics/db/aggregation.txt

```diff
@@ -632,3 +637,34 @@ aggregate that author count, referencing the annotation field:
     >>> from django.db.models import Avg, Count
     >>> Book.objects.annotate(num_authors=Count("authors")).aggregate(Avg("num_authors"))
     {'num_authors__avg': 1.66}
+
+Handling empty querysets using aggregate ``default`` 
+----------------------------------------------------
+
+When an aggregate function is used with an empty ``QuerySet`` or grouping,
+None is returned as a default, this is because aggregate functions result 
+in NULL when used with an empty result set. The optional ``default`` argument 
+for aggregate functions allows specifying a value to use as a default when the 
+executed query returns no rows to perform an aggregation. Some aggregate functions 
+do not support the ``default`` argument because they already handle this, e.g. 
+``Count`` returns ``0``.
+
+For example, say you wanted to get the total price of all books containing 
+"web" in name. Without the ``default`` argument, if there are no books 
+returned by the query, and there are no rows to compute the aggregate 
+``Sum``, None is returned as the computed aggregate.
+
+.. code-block:: pycon
+
+    >>> from django.db.models import Sum
+    >>> Book.objects.filter(name__contains="Web").aggregate(Sum("price"))
```

Comment:
In the paragraph above the example you have `"web"` and in the example you have `"Web"` (uppercased first letter).

```suggestion
    >>> Book.objects.filter(name__contains="web").aggregate(Sum("price"))
```